code/game/objects/items/theft_items.dm

```diff
@@ -49,14 +48,50 @@
 	icon = 'icons/obj/nuke_tools.dmi'
 	icon_state = "core_container_empty"
 	item_state = "metal"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //Don't want people trying to break it open with acid, then destroying the core.
 	var/obj/item/nuke_core/plutonium/core
+	var/dented = FALSE
+	var/cracked = FALSE
 
 /obj/item/nuke_core_container/Destroy()
 	QDEL_NULL(core)
 	return ..()
 
+/obj/item/nuke_core_container/ex_act(severity)
+	switch(severity)
+		if(1)
+			if(!cracked)
+				crack_open()
+			return
+		if(2)
+			if(!dented)
+				dented = TRUE
+			return
+
+		if(3)
+			return
+
+/obj/item/nuke_core_container/examine(mob/user)
+	. = ..()
+	if(!cracked)
+		. += "Fine print on the box reads \"Cybersun Industries secure container, guaranteed thermite proof, assistant proof, and explosive resistant.\""
+	if(dented && !cracked)
+		. += "<span class='notice'>[src] looks dented. Perhaps a bigger explosion may break it."
+	if(cracked)
+		. += "<span class='warning'>It is broken, and can no longer store objects safely.</span>"
+
+/obj/item/nuke_core_container/attack_hand(mob/user)
+	if(cracked && core)
+		core.add_fingerprint(user)
+		user.put_in_active_hand(core)
+		core = null
+		icon_state = "core_container_cracked_empty"
```

Comment:
I'd put this code in an `unload` proc defined on `/obj/item/nuke_core_container` level. Then override it for the child or something

Reply:
Can do